.github/workflows/test.yml

```diff
@@ -1,5 +1,5 @@
 name: Tests
-on: push
+on: [push, pull_request]
```

Comment:
We do not need that change anymore, especially since you showed that PR from branches in forks cannot use the license key.

Reply:
Right, I will revert that change then.